tetrateci/create_aks_cluster.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+CLUSTER_NAME="test-istio-$GITHUB_SHA"
```

Comment:
Here and elsewhere, we also need to include the cluster version in the name. If we have matrix strategy and spin up multiple clusters for the same SHA. Use SHORT SHA for the cluster name.